specification/fabric/Microsoft.Fabric.Management/models.tsp

```diff
@@ -0,0 +1,176 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using Azure.Core;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+using OpenAPI;
+
+namespace Microsoft.Fabric;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+@doc("The Microsoft Fabric capacity properties.")
+model FabricCapacityProperties {
+  @doc("The current deployment state of Microsoft Fabric resource. The provisioningState is to indicate states for resource provisioning.")
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+
+  @doc("The current state of Microsoft Fabric resource. The state is to indicate more states outside of resource provisioning.")
+  @visibility("read")
+  state: State;
+
+  @doc("The capacity administration")
+  administration: CapacityAdministration;
+}
+
+@doc("Represents the SKU name and Azure pricing tier for Microsoft Fabric capacity resource.")
+model RpSku {
+  @doc("The name of the SKU level.")
+  name: string;
+
+  @doc("The name of the Azure pricing tier to which the SKU applies.")
+  tier: RpSkuTier;
+}
+
+@doc("The name of the Azure pricing tier to which the SKU applies.")
+union RpSkuTier {
```

Comment:
Why is this a union?

Reply:
removed "string"

Reply:
becuase enums should be defined as union
![image](https://github.com/Azure/azure-rest-api-specs/assets/127952241/afc40712-2e79-4f40-95c5-dc9263a8ecf7)

and when using union you must specify the base type
![image](https://github.com/Azure/azure-rest-api-specs/assets/127952241/bf75e491-c357-4c7c-8c63-2e8627c24549)